README.md

```diff
@@ -0,0 +1,138 @@
+that only includes conda and its dependencies.
+
+
+## Getting Started
+
+If you install the Anaconda Distribution, you will already have hundreds of packages
+installed.  You can see what packages are installed by running
+
+```bash
+$ conda list
+```
+
+to see all the packages that are available, use
+
+```bash
+$ conda search
+```
+
+and to install a package, use
+
+```bash
+$ conda install <package-name>
+```
+
+The real power of conda comes from its ability to manage environments.
+In conda, an environment can be thought of as a completely separate installation.
+Conda installs packages into environments efficiently using [hard links](https://en.wikipedia.org/wiki/Hard_link) by default when it is possible, so
+environments are space efficient, and take seconds to create.
+
+The default environment, which `conda` itself is installed into is called
+`base`.  To create another environment, use the `conda create`
+command. For instance, to create an environment with the IPython notebook and
+NumPy 1.6, which is older than the version that comes with Anaconda by
+default, you would run:
+
+```bash
+$ conda create -n numpy16 ipython-notebook numpy=1.6
+```
+
+This creates an environment called `numpy16` with the latest version of
+the IPython notebook, NumPy 1.6, and their dependencies.
+
+We can now activate this environment, use
+
+```bash
+$ conda activate numpy16
+```
+
+This puts the bin directory of the `numpy16` environment in the front of the
+`PATH`, and sets it as the default environment for all subsequent conda commands.
+
+To go back to the base environment, use
+
+```bash
+$ conda deactivate
+```
+
+## Building Your Own Packages
+
+You can easily build your own packages for conda, and upload them
+to [anaconda.org](https://anaconda.org), a free service for hosting
+packages for conda, as well as other package managers.
+To build a package, create a recipe. Package building documentation is available
+[here](https://docs.conda.io/projects/conda-build/en/latest/).
+See [AnacondaRecipes](https://github.com/AnacondaRecipes) for the recipes that make up the Anaconda Distribution and `defaults` channel.
+[Conda-forge](https://conda-forge.org/feedstocks/) and [Bioconda](https://github.com/bioconda/bioconda-recipes) are community-driven conda-based distributions.
+
+To upload to anaconda.org, create an account.  Then, install the
+anaconda-client and login
+
+```bash
+$ conda install anaconda-client
+$ anaconda login
+```
+
+Then, after you build your recipe
+
+```bash
+$ conda build <recipe-dir>
+```
+
+you will be prompted to upload to anaconda.org.
+
+To add your anaconda.org channel, or the channel of others to conda so
+that `conda install` will find and install their packages, run
+
+```bash
+$ conda config --add channels https://conda.anaconda.org/username
+```
+
+(replacing `username` with the user name of the person whose channel you want
```

Comment:
I believe the second "username" should still be one word.

Reply:
```suggestion
(replacing `username` with the username of the person whose channel you want
```